src/App.jsx

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import './App.scss';
+import { Person } from './components/Person/Person';
```

Comment:
Ensure that the `Person` component is correctly defined and exported in `./components/Person/Person`. If it's not defined, this will cause an error when rendering the `App` component.